.babelrc

```diff
@@ -1,7 +1,19 @@
 {
-    "presets": ["es2015", "flow"],
+    "presets": [
+      ["es2015", {
+          "modules": false
+      }],
+      "flow"
+    ],
     "plugins": [
         "transform-runtime",
         "transform-class-properties"
-    ]
+    ],
+    "env": {
+      "test": {
+        "plugins": [
+            "transform-es2015-modules-commonjs"
```

Comment:
This is for Jest, as it [doesn't support ES6 modules](https://facebook.github.io/jest/docs/en/getting-started.html).
In Babel 6, env configurations doesn't overrides configuration, rather are merged to configuration(babel/babel#5276).